server/src/test/java/com/octopus/teamcity/opentelemetry/server/TestBuildStorageManager.java

```diff
@@ -0,0 +1,17 @@
+package com.octopus.teamcity.opentelemetry.server;
+
+import jetbrains.buildServer.serverSide.SBuild;
+import jetbrains.buildServer.serverSide.SRunningBuild;
+import org.jetbrains.annotations.Nullable;
+
+public class TestBuildStorageManager implements BuildStorageManager {
+    @Nullable
+    @Override
+    public String getTraceId(SBuild build) {
+        return "1234";
```

Comment:
I'm unsure if this is an accurate test 😂

Reply:
It's not a test, it's a mock implementation of the BuildStorageManager.
(I feel I'm holding things wrong if you're assuming things are tests?)